packages/relative-values/models/src/animals.ts

```diff
@@ -0,0 +1,617 @@
+mean_num_chickens_per_year_per_commitment = 5k to 20k 
+// this: <https://thehumaneleague.org/article/victory-toridoll-releases-a-global-cage-free-commitment?ms=c_blog>
+// mentions 40k, but also seems to be the most impressive one for 2022 (?)
+frequency_promises_are_fulfilled = 0.4 to 0.9
+average_duration_fulfilled_commitments = 1 to 10 // years. Could be more?
+average_reduction_in_suffering_of_commitment_per_chicken = 0.05 to 0.3 // pulled out of thin air/very subjective estimate here.
+
+value_owa_2022 = num_corporate_commitments * 
+  mean_num_chickens_per_year_per_commitment *
+  frequency_promises_are_fulfilled *
+  average_duration_fulfilled_commitments *
+  average_reduction_in_suffering_of_commitment_per_chicken *
+  value_tortured_chicken_year_in_human_qalys *
+  chicken_estimates.lifetime
+
+owa_item = {
+  name: "Open Wing Alliance",
+  year: 2022, 
+  slug: "open_wing_alliance",
+  value: value_owa_2022 -> ss
+}
+
+/* Beyond Meat */
+
+beyond_meat_price_per_pound = 5 to 12
+beyond_meat_revenues = 375M to 415M
+beyond_meat_pounds_sold = beyond_meat_revenues / beyond_meat_price_per_pound
+
+pounds_of_meat_per_cow = weight_cow * kilograms_in_pounds *  non_wastage_proportion_cow
+cows_saved_if_all_meat_was_counterfactual = beyond_meat_pounds_sold / pounds_of_meat_per_cow
+
+chance_purchase_was_counterfactual = 0.05 to 0.25 // should eventually be a beta
+// I imagine that most people would have chosen a vegetarian alternative
+// but also that having tasty meat replacements makes it easier to
+// keep being a vegetarian.
+// Very uncertain, though
+
+value_beyond_meat_year =  cows_saved_if_all_meat_was_counterfactual * 
+  chance_purchase_was_counterfactual * 
+  cow_estimates.lifetime *
+  cow_estimates.value_year_in_human_qalys
+
+beyond_meat_item = {
+  name: "Beyond Meat",
+  year: 2022, 
+  slug: "beyond_meat",
+  value: value_beyond_meat_year -> ss
+}
+
+/* ACE Top Charities */ 
+
+/* From here on, estimates of ACE's top charities are constructed with reference to the above. 
+I consider them to be pretty fake (i.e., not-mechanistic), 
+though they could eventually become something more fleshed out.
+Maybe we could  */
+
+value_reference_fish_org = fwi_item.value 
+value_reference_chicken_org = owa_item.value
+value_reference_cow_org = beyond_meat_item.value / (10 to 1k) // beyond meat seems significantly more scaled up than the avg 
+value_reference_top_animal_org = mixture(
+  [value_reference_cow_org, value_reference_chicken_org, value_reference_chicken_org], 
+  [2/3, 1/3, 1/3]
+) -> ss
+
+// Add uncertainty
+values_reference_orgs = [value_reference_fish_org, value_reference_chicken_org, value_reference_cow_org]
+highly_uncertain_org = List.map(values_reference_orgs, {|dist| 
+ lower = inv(dist, 0.05)
+ upper = inv(dist, 0.95)
+ new_dist = normal({p5: lower*100, p95: upper*100}) 
+ // ^ only works because low < 0 and high>0
+ new_dist
+}) -> mixture
+add_uncertainty(dist) = mixture([dist, highly_uncertain_org], [95, 5])
```

Comment:
I like this function. Not sure about the specifics, but the general idea.

I'd also note that you can do a yMap to get a similar effect, with long tails.

Reply:
Thanks.

> I'd also note that you can do a yMap to get a similar effect, with long tails.

Nice, will keep in mind.